src/Google.Cloud.Diagnostics.Debug/Google.Cloud.Diagnostics.Debug/BreakpointReadActionServer.cs

```diff
@@ -68,8 +68,8 @@ internal override void MainAction()
             else
             {
                 breakpoint.IsFinalState = true;
+                _debuggerClient.UpdateBreakpoint(breakpoint);
```

Comment:
Shouldn't we still update the breakpoint?  This could give error messages or updates to the line it was set on.